packages/gradient_boosting_model/gradient_boosting_model/datasets/data_description.txt

```diff
@@ -0,0 +1,523 @@
+MSSubClass: Identifies the type of dwelling involved in the sale.	
```

Comment:
I am also uncertain as to wheter I would include this in the repo. This information is together with the data set and can be viewed at the right source. See my comment on the data set re-hosting. I would remove from the repo. Copyright potential issues.

Reply:
removed